modules/nf-core/last/train/tests/main.nf.test.snap

```diff
@@ -8,7 +8,7 @@
                             "id": "contigs",
                             "single_end": false
                         },
-                        "contigs.STUB.train:md5,d41d8cd98f00b204e9800998ecf8427e"
+                        "contigs..des.train:md5,d41d8cd98f00b204e9800998ecf8427e"
```

Comment:
I just replace the `INDEX_NAME=STUB` in the stub by the same command from the script:
`INDEX_NAME=\$(basename \$(ls $index/*.des) .des)`

Reply:
Since the stub does not receive a `.des` file, ` ls $index/*.des` returns nothing and the basename command becomes `basename .des`, which returns `.des`.